workers/common/src/main/kotlin/common/env/config/EnvironmentConfigLoader.kt

```diff
@@ -110,7 +110,17 @@ class EnvironmentConfigLoader(
      * content. If not file path is provided, the default path `.ort.env.yml` is used. Syntactic errors in the file
      * cause exceptions to be thrown. Semantic errors are handled according to the `strict` flag.
      */
-    fun parse(repositoryFolder: File, environmentConfigPath: String? = null): EnvironmentConfig {
+    fun resolveAndParse(repositoryFolder: File, environmentConfigPath: String? = null): EnvironmentConfig =
+        resolveEnvironmentFile(repositoryFolder, environmentConfigPath)?.let { config ->
```

Comment:
A more consistent naming would help with readability, e.g. consistently use `environmentFile` or `environmentConfigFile` instead of `environmentFile`, `config`, and `environmentConfigFile` when they refer to the same thing. Also the `parse` function should be renamed accordingly to `parseEnvironment(Config)File`.